src/FakeItEasy.Net35/Compatibility/ConcurrentQueue.cs

```diff
@@ -0,0 +1,34 @@
+namespace FakeItEasy

```

Comment:
> if I use System.Collection.Concurrent, the compiler can't tell it from the real one when it builds the .NET 3.5 unit tests.

Isn't it just because `FakeItEasy.Net35.Tests` actually targets .NET 4.5? It *should* target 3.5... There are just two small gotchas:
- it references `FakeItEasy.Tests`, which targets 4.5. Easily fixed by just linking the source files of the helper classes
- it uses FluentAssertions, which requires at least 4.0. But there are very few tests in `FakeItEasy.Net35.Tests`, so it would probably be OK to not use FluentAssertions for those tests.